Extension/src/LanguageServer/client.ts

```diff
@@ -21,6 +21,7 @@ import { createProtocolFilter } from './protocolFilter';
 import { DataBinding } from './dataBinding';
 import minimatch = require("minimatch");
 import * as logger from '../logger';
+import { deactivate } from './extension';
```

Comment:
This should not be imported here.  Only vscode should ever call this method.

Reply:
That snuck in via autocomplete. Removed + noted for the future